operations/iscsi_sbps/iscsi_sbps.md

```diff
@@ -400,20 +418,31 @@ Please refer [iSCSI Metrics](https://github.com/Cray-HPE/sbps-marshal/blob/main/
 
 ## Steps to switch from DVS to iSCSI
 
+This involves this sequence of operations:
+
 * Disable DVS
 * Uninstall CPS
 * Enable iSCSI SBPS
 
-To disable DVS please refer:
-<https://github.hpe.com/hpe/hpc-shasta-os-uss-docs/blob/master/dac-assets/documents/operations/DVS_and_SBPS.md>
+To perform these operations, please refer the section `DVS and SBPS` documented in the publication:
+`HPE Cray Supercomputing User Services Software Administration Guide: CSM on HPE Cray Supercomputing EX Systems (S-8063)`.
+
+## Steps to continue using DVS based projection
 
-To uninstall CPS services please refer:
-<https://github.hpe.com/hpe/hpc-shasta-os-uss-docs/blob/master/dac-assets/documents/operations/Uninstall_CPS.md>
+If a user wants to continue using DVS, then during the BOS session template creation (manually using `cray bos` command)
```

Comment:
The "sat bootprep" files for compute and UAN have SBPS enabled by default, but include the DVS stanzas in the file but commented out.   Not sure if we'd want to mention that.